public/api-documentation/swagger.json

```diff
@@ -401,6 +441,32 @@
         }
       }
     },
+    "SubmitSignature": {
+      "type": "object",
+      "properties": {
+        "answer": {
+          "type": "string"
+        },
+        "question_id": {
+          "type": "string"
+        },
+        "token": {
+          "type": "string"
+        },
+        "user_id": {
+          "description": "CDX user id",
+          "type": "string"
+        }
+      }
+    },
+    "SubmitTransaction": {
```

Comment:
I had split up the "new" API from removing the "old" API as separate tasks. If it makes it easier to follow, I can make the "new" implementation take the place of the "old" signature API in this step. It certainly makes naming simpler.